app/src/main/java/org/p2p/wallet/auth/interactor/AuthLogoutInteractor.kt

```diff
@@ -45,7 +46,7 @@ class AuthLogoutInteractor(
             analytics.setUserId(null)
 
             updatesManager.stop()
-            sharedPreferences.edit { clear() }
+            userDataStore.clear()
```

Comment:
does it clean up all the values inside the data store? including device share and other stuff that were in account_prefs.xml?